teamcity-rest-client-api/src/main/kotlin/org/jetbrains/teamcity/rest/common.kt

```diff
@@ -53,6 +53,13 @@ interface BuildLocatorSettings<Self : BuildLocatorSettings<Self>> {
     fun onlyCanceled(): Self
 
     fun withStatus(status: BuildStatus): Self
+
+    /**
+     * By default failedToStart builds are not returned
+     */
+    fun includeFailedToStart(): Self
+    fun withFailedToStart(flag: Boolean): Self
```

Comment:
I would rather have `onlyFailedToStart(): Self` here, similar to what we have with cancelled. From my perspective, the whole thing is very similar to cancelled in terms of a user experience: usually there is no need to have them in response, but when they are needed it's likely some kind of an alert or investigation, and then `onlyFailedToStart()` makes sense.

Reply:
Implemented same way as cancelled